### DISCUSSION TITLE
Kunena Forum aurelia

### DISCUSSION THREAD

**Discussion by Leen29** [Kunena/Kunena-Forum#9564] [Ideas]
I wonder if it is possible to get into the
/components/com_kunena/template/aurelia/assets/scss/custom.scss to skip this with upgrade
now the created cusom.scss data is lost during Upgrade
6.2.0-BETA2 2023-09-26

**Comment by xillibit** [comment]
Hello,

If i'am remeber right it has been fixed since, do-you have still the issue with the Kunena 6.2.4 ?

**Reply by Leen29** [reply]
Hello,
ai i forget this post
  no i have this  issue  with Kunena 6.2.4 any more
  Thank You
